test/inference/mh.jl

```diff
@@ -8,7 +8,7 @@ include(dir*"/test/test_utils/AllUtils.jl")
         Random.seed!(0)
         N = 500
         s1 = MH(
-            (:s, GKernel(3.0)),
+            (:s, InverseGamma(2,3)),
             (:m, GKernel(3.0)))
```

Comment:
Is `GKernel` equivalent with `Normal`? If so, maybe consider replacing it with `Normal`.

Reply:
`GKernel` returns a function that you can draw proposals from:

https://github.com/TuringLang/Turing.jl/blob/97e9eb534f68e45b9558bab5e4f5ab047c920f20/test/test_utils/testing_functions.jl#L3

So this is a random-walk style proposal. I left this method in for backwards compatibility, as you can replicate this behavior in a more direct way by constructing

```julia
MH(:m:  => AMH.Proposal(RandomWalk(), Normal(0,3), . . .)
```